apps/src/applab/commands.js

```diff
@@ -866,6 +874,13 @@ applabCommands.setImageURL = function (opts) {
   var element = document.getElementById(opts.elementId);
   if (divApplab.contains(element) && element.tagName === 'IMG') {
     element.src = Applab.maybeAddAssetPathPrefix(opts.src);
+
+    if (!ensureCached[element.src]) {
+      ensureCached[element.src] = true;
+      var img = new Image();
+      img.src = element.src;
```

Comment:
Is there any risk of this causing a memory leak?  We don't hold a reference to `img` anywhere so it should be eligible for GC as soon as it goes out of scope.